### DISCUSSION TITLE
Guidance on adding new chemicals to database

### DISCUSSION THREAD

**Discussion by bryceconduitEQ** [equinor/neqsim#1081] [General]
Hi,

I just getting started with Neqsim via the python interface. I see there are already some amines programmed into the databases. For planning purposes at this stage I'm trying to work out what would be required to add a new amino acid salt to the database - I would appreciate any guidance on this/literature/experimental data that is likely to be required.

https://github.com/equinor/neqsim/tree/master/src/main/resources/data

Thanks,

Bryce

**Comment by EvenSol** [comment]
Hi.
Please see [this notebook](https://colab.research.google.com/drive/1vvaaooaD0K8dVh-h3OP2SzOl5RVhDVNS#scrollTo=XfChsVrQ1H6j) for an example of how to add a potassium salt of sarcosine (KSAR). This illustrates the general way that chemicals and reactions are added in NeqSim.

Thes next step would be to fit parameters as was done in this [thesis](https://ntnuopen.ntnu.no/ntnu-xmlui/handle/11250/231326).

**Reply by bryceconduitEQ** [reply]
Great many thanks for this. 
Bryce
